src/main/java/com/spring/address/db/models/Address.java

```diff
@@ -0,0 +1,26 @@
+package com.spring.address.db.models;
```

Comment:
Weird project structure. You should split files into packages by layers of logic. Restructure the whole project. I suggest using the following structure:
```
.
└── com.spring/
    ├── api/
    │   ├── controller/
    │   │   ├── UserController.java
    │   │   └── StudentController.java
    │   ├── dto/
    │   │   ├── UserDto.java
    │   │   └── StudentDto.java
    │   └── transformer/
    │       ├── UserDtoTransformer.java
    │       └── StudentDtoTransformer.java
    ├── db/
    │   ├── entity/
    │   │   ├── student/
    │   │   │   └── Student.java
    │   │   └── user/
    │   │       └── User.java
    │   ├── configuration/
    │   │   ├── AddressDBConfiguration.java
    │   │   └── UserDBConfiguration.java
    │   └── repository/
    │       ├── UserRepository.java
    │       └── StudentRepository.java
    ├── service/
    │   ├── UserService.java
    │   └── StudentService.java
    └── UserApplication.java
```